content/docs/setup/kubernetes/additional-setup/config-profiles/index.md

```diff
@@ -53,12 +53,17 @@ The components marked as **X** are installed within each profile:
 Some profiles have an authentication variant, with `-auth` appended to the name, which adds the following
 security features to the profile:
 
-| | default | demo | minimal | sds |
+| | default | demo | minimal | sds-auth |
 | --- | --- | --- | --- | --- |
 | Control Plane Security | | X | | |
 | Strict Mutual TLS | | X | | X |
 | SDS | | | | X |
 
+{{< tip >}}
```

Comment:
preceeed the table with the tip please.

Reply:
Putting the tip before the table looks a little strange to my eye in the preview. Anything we can do about this? Or does it look as expected?